Vendor Kubernetes 1.16.0-rc.2

saschagrunert opened the pull request:
Vendoring the latest RC to see how the CI behaves and adding the new `Status.Network.AdditionalIps` fields for pods.

Random-Liu commented:
@saschagrunert Is this WIP?

And what command did you run to update the go mod? Just for my education.

saschagrunert commented:
> @saschagrunert Is this WIP?
> 
> And what command did you run to update the go mod? Just for my education.

I updated it to RC2 and removed the WIP as discussed.

For the general update, I manually edited go.mod to contain the same commit (the release candidate) in the `replace` section at the bottom. Then running `./hack/verify-go-modules.sh` should vendor in the changes. :)
